coderd/database/migrations/000153_agent_default_apps.down.sql

```diff
@@ -1,5 +1,2 @@
-BEGIN;
 ALTER TABLE workspace_agents DROP COLUMN display_apps;
-DROP TYPE display_app; 
-COMMIT;
-
+DROP TYPE display_app;
```

Comment:
Does dropping a type also potentially have the same issue as adding a type?

Reply:
Since it's dropping the entire type it doesn't have the same consequences as `ADD VALUE`.